### PR TITLE
Sample MSMQ to Azure Service Bus using the transport bridge

### PR DESCRIPTION
**Pull request by TravisNickels**

### CODE REVIEW

File: samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/Bridge/Program.cs [hunk 1]
```diff
@@ -1,69 +1,48 @@
...
-
-
-        #endregion
+        await Host.CreateDefaultBuilder()
+            .UseNServiceBusBridge((ctx, bridgeConfiguration) =>
+            {
+                var asbTransport = new AzureServiceBusTransport(connectionString)
+                {
+                    //TransportTransactionMode = TransportTransactionMode.ReceiveOnly
+                    TransportTransactionMode = TransportTransactionMode.SendsAtomicWithReceive
```
Comment (andreasohlund) [inline samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/Bridge/Program.cs:24]: No need for this, the bridge will default to receiveonly if one of the transports can't do DTC

File: samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/AsbEndpoint/Program.cs [hunk 2]
```diff
@@ -13,37 +13,20 @@ static async Task Main()
...
-        var transport = endpointConfiguration.UseTransport<AzureServiceBusTransport>();
-        var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus.ConnectionString");
+        var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
         if (string.IsNullOrWhiteSpace(connectionString))
         {
             throw new Exception("Could not read the 'AzureServiceBus.ConnectionString' environment variable. Check the sample prerequisites.");
         }
-        transport.ConnectionString(connectionString);
-
-        #region connect-asb-side-of-bridge
```
Comment (andreasohlund) [inline samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/AsbEndpoint/Program.cs:28]: https://github.com/Particular/docs.particular.net/blob/master/samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/sample.md needs to be updated to reflect the changes made

File: samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/Bridge/Program.cs [hunk 3]
```diff
@@ -1,69 +1,39 @@
...
+            {
+                var asbBridgeTransport = new BridgeTransport(new AzureServiceBusTransport(connectionString));
+                asbBridgeTransport.AutoCreateQueues = true;
+                var asbEndpoint = new BridgeEndpoint("Samples.Azure.ServiceBus.AsbEndpoint");
+                asbEndpoint.RegisterPublisher<MyEvent>("Samples.Azure.ServiceBus.MsmqEndpoint");
+                asbBridgeTransport.HasEndpoint(asbEndpoint);
+
+                var msmqBridgeTransport = new BridgeTransport(new MsmqTransport());
+                msmqBridgeTransport.AutoCreateQueues = true;
+                var msmqEndpoint = new BridgeEndpoint("Samples.Azure.ServiceBus.MsmqEndpoint", $"Samples.Azure.ServiceBus.MsmqEndpoint@{Environment.MachineName}");
```
Comment (andreasohlund) [inline samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/Bridge/Program.cs:29]: Is setting the machine name needed? (our acceptance tests don't need to do this?)

Comment (TravisNickels) [reply samples/azure-service-bus-netstandard/azure-service-bus-msmq-bridge/ASBS_1/Bridge/Program.cs:29]: For these samples, if we don't include the machine name, the MSMQ endpoint receives the command from the ASB endpoint, but the subsequent publish event from the MSMQ endpoint is never handled by the ASB endpoint.

**Review by andreasohlund — APPROVED** [review]

**Review by andreasohlund — APPROVED** [review]

**Pull request merged**